preset/variables.scss

```diff
@@ -139,7 +139,7 @@ $dialog-border-radius: 8px;
 $input-label-letter-spacing: normal;
 
 // v-text-field
-$text-field-outlined-fieldset-border: 2px solid map-get($grey, 'lighten1');
+$text-field-outlined-fieldset-border-width: 2px;
```

Comment:
Does this change all text-field borders to 2px and is that what we're expecting:

<img width="660" alt="Storypark 2020-07-28 15-21-25" src="https://user-images.githubusercontent.com/1263717/88615561-4fe9cf80-d0e6-11ea-8ded-5293f9c72b0f.png">

Reply:
yeah i think that will change all text-fields with the outlined prop

Reply:
Yea—that doesn't seem right. The PR description is about error outlines not the border size. Is this a change we're expecting to make? It doesn't seem match [the designs](https://projects.invisionapp.com/share/R7WY9NH4D82#/screens/417071391), has there been a different discussion though?